src/main/js/electron/index.py

```diff
@@ -13,212 +14,479 @@
+           visible = True
+        ),
+        error_y = dict(
+           type = 'data',
+           array = newError_rate,
+           visible = True
         )
-
-        layout=dict(
-                     title='',
-                     xaxis=dict(
-                         title='Time',
-                         rangeslider=dict(),
-                         titlefont=dict(
-                         family='Courier New, monospace',
-                         size=18,
-                         color='#7f7f7f'
-                            )
-                     ),
-                     yaxis=dict(
-                         title='Count Rate',
-                         titlefont=dict(
-                         family='Courier New, monospace',
-                         size=18,
-                         #range=[int(start), int(end)],
-                         #range=[10,20],
-                         color='#7f7f7f'
-                          )
-                     )
+    )
+
+    layout1 = dict(
+         title = '',
+         xaxis = dict(
+             title = 'Time',
+             #range = [start_time, end_time],
+             titlefont = dict(
+                 family = 'Courier New, monospace',
+                 size = 18,
+                 color = '#7f7f7f'
+            )
+         ),
+         yaxis=dict(
+             title='Count Rate',
+             #range=[start_count, end_count],
+             titlefont = dict(
+                 family = 'Courier New, monospace',
+                 size = 18,
+                 color = '#7f7f7f'
+             )
+         ),
+        dragmode='select'
+
+    )
+    trace2 = dict(
+        type = 'scatter',
+        x = newcolor1,
+        y = newcolor2,
+        error_x = dict(
+           type = 'data',
+           array = newError_color1,
+           visible = True
+        ),
+        error_y = dict(
+           type = 'data',
+           array = newError_color2,
+           visible = True
+        )
+    )
+
+    layout2 = dict(
+         title = '',
+         xaxis = dict(
+             title = 'Color1',
+             #range = [start_color1, end_color1],
+             titlefont = dict(
+                 family = 'Courier New, monospace',
+                 size = 18,
+                 color = '#7f7f7f'
             )
+         ),
+         yaxis=dict(
+             title='Color2',
+             #range=[start_color2, end_color2],
+             titlefont = dict(
+                 family = 'Courier New, monospace',
+                 size = 18,
+                 color = '#7f7f7f'
+             )
+         ),
+         dragmode='select'
+    )
+    fig=[]
+    fig1 = dict(data = [trace1], layout = layout1)
+    fig2 = dict(data = [trace2], layout = layout2)
+    fig.append(fig1);
+    fig.append(fig2);
+
+    return fig
+
 
+
+def upload_file2(filename=None, start_time=None, end_time=None, start_count=None, end_count=None,start_color1=None,end_color1=None,start_color2=None,end_color2=None):
```

Comment:
Please use method names that are descriptive of what the method does. If it's hard to do that, then please consider moving code around so that methods have a clear purpose and can easily be named.